Lib/test/test_asyncio/test_subprocess.py

```diff
@@ -460,6 +460,10 @@ async def cancel_make_transport():
             test_utils.run_briefly(self.loop)
 
     def test_close_kill_running(self):
+        if sys.platform != "win32":
+            if isinstance(asyncio.get_child_watcher(),
```

Comment:
Can you just check `self.Watcher` directly?

Reply:
Yep, that works as well, and it would save us from having to use `isinstance()`. Thanks for the suggestion.

Since it's a bit shorter now, I'll also condense it to `if sys.platform != "win32" and self.Watcher == asyncio.MultiLoopChildWatcher`.